terraform/lambdas.tf

```diff
@@ -82,7 +82,7 @@ resource "aws_lambda_function" "raster_analysis_fanout" {
   tags             = local.tags
   layers           = [
     module.lambda_layers.raster_analysis_arn,
-    data.terraform_remote_state.lambda-layers.outputs.py310_shapely_185_arn,
+    data.terraform_remote_state.lambda-layers.outputs.py310_shapely_no_numpy_arn,
```

Comment:
** Do you need to add in the numpy layer here, since you're using a shapely layer with no numpy?

Reply:
Good point, I think you're right!

Reply:
Fixed!